ictv/pages/channel_page.py

```diff
@@ -262,7 +264,7 @@ def POST(self, channel_id, channel):
             if channel is not None and channel.enabled:
                 form.enabled = 'on'
         store_form(form)
-        raise web.seeother('/channels/%d' % channel.id)
+        resp.seeother('/channels/config/%d' % channel.id)
```

Comment:
Was changing the path to ``.../config/...`` necessary ?

Reply:
@l9kd1 did it because it was conflicting with the plugins sub-apps that have similar routes. 
The routing is managed slightly differently in flask with the way we implemented the sub-apps.

So probably not necessary but simpler